### DISCUSSION TITLE
Localization keyword

### DISCUSSION THREAD

**Discussion by heischo** [dotnet/roslyn#70372] [Ideas]
I think an extra keyword for localization would make a lot of things easier: 
`localizationRef("localizationName") ` 
An optional second parameter can change the source of the language setting, while the default is the CurrentCulture of the thread:
`localizationRef("localizationName", cultureProvider) `

**Comment by CyrusNajmabadi** [comment]
@heischo you could accomplish that by defining such a method yourself, globally importing it, and then using it wherever you want :)

**Reply by heischo** [reply]
@CyrusNajmabadi thanks! What do you mean by 'globally importing it'? I've never used that, and when I try to find information, I only get presented the global using in C# 10. I remember reading something about this a long time ago, but I can't figure out how to look for it specifically.

**Reply by CyrusNajmabadi** [reply]
`global using static MyClass;`

Now you access the static members of MyClass anywhere.  So just define your helper in that, and you're golden.

**Reply by heischo** [reply]
@CyrusNajmabadi  okay, thank you. I thought it was something "more complicated", like using the Global Assembly Cache (that I never touched) :)
